benchmark_runner/common/oc/oc.py

```diff
@@ -692,6 +714,17 @@ def wait_for_pod_completed(self, label: str, workload: str = '', label_uuid: boo
         except Exception as err:
             raise PodNotCompletedTimeout(workload=workload)
 
+    def describe_vmi(self, vm_name: str, namespace: str):
+        """
+        This method describe vmi into log
+        :param vm_name: vm name with uuid
+        :param namespace: namespace
+        :return: output_filename
+        """
+        output_filename = os.path.join(self._run_artifacts, f'describe-{vm_name}')
+        self.run(f"{self.__cli} describe vmi -n {namespace} {vm_name} > {output_filename} ")
```

Comment:
Same quote and trailing whitespace as above.

Reply:
I have added it only to '{output_filename}', for 'oc' it will cause syntax's issue and also we pass this value.

Reply:
See above.